.github/workflows/claude-readme-sync.yml

```
@@ -0,0 +1,60 @@
+            
+            ## Important Notes
+            - Keep all code examples exactly as they are
+            - Preserve all image links and badges
+            - Maintain the same document structure
+            - Ensure technical terms are translated appropriately
+            - Keep the tone professional and clear
+            
+            Please complete this translation task and save the result to the appropriate file.
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)